smaps/src/main/java/com/google/smaps/SearchAddress.java

```diff
@@ -34,8 +34,11 @@
 @WebServlet(name = "SearchAddress", value = "/searchaddress")
 public class SearchAddress extends HttpServlet {
   // Stores the String representation of the hex address as the user entered it.
+  static String originalAddress;
+  // Stores the originalAddress as lowercase to use for parsing.
   static String address;
```

Comment:
Is there a reason to have all of these be class variables? It seems like you could just have address be a variable in the doPost method.

Reply:
I thought about having address just be in doPost, but I think I want to use the reformatted version of the address that matches the memory map format to refill the search box after the user clicks "go," so I send address, and not originalAddress, to the javascript in doPost.